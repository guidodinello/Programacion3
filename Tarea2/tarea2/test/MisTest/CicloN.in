crear_personas 10
leer_lista_muertes 1 2 2 3 3 4 4 5 5 6 6 7 7 8 8 9 9 1 .
leer_lista_convivencias .
existe_cronologia
Fin
