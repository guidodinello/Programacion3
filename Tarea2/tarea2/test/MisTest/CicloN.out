1>2>3>4>No hay una cronologÃ­a consistente.
5>Fin.
